src/Engines/constants.ts

```diff
@@ -57,6 +57,11 @@ export class Constants {
      * Alpha will be set to DST ALPHA
      */
     public static readonly ALPHA_EXCLUSION = 16;
+    /**
+     * Defines that alpha blending is SRC * SRC ALPHA + DST * (1 - SRC ALPHA)
+     * Alpha will be set to SRC ALPHA + (1 - SRC ALPHA) * DST ALPHA

```

Comment:
I'm not sure of the intent of this blend mode so please ignore these comments if they're out of place:

Alpha accumulation should use "over" blending if blending SRC over (i.e. in front) of DST. This is just the current `ALPHA_COMBINE`.
Alpha accumulation should use "under" blending if blending SRC under (i.e. behind) of DST. This would be `SRC * (1 - DST_ALPHA) + DST * (DST_ALPHA)`

Additionally, this alpha blend equation looks odd to me since each layer is increasing the alpha of the destination by more than SRC_ALPHA.

Reply:
Actually code comes from [this example](https://github.com/tsherif/webgl2examples/blob/master/oit-dual-depth-peeling.html) but I gave it a further look because it seemed odd to me too that we don't use DST ALPHA in the color part of the blend.

So it turns out this is for back blending. Front blending is done manually in the shader [here](https://github.com/BabylonJS/Babylon.js/blob/b54bb0366da18e2c6c3dc3b8ac65f6af189b9b02/src/Shaders/default.fragment.fx#L468). And the alpha accumulation of the value only makes sense at the very end, when front peels and back peels meet to perform the final blending. At this moment we need the total alpha accumulation of the back peels to perform one final blending of front over back ([here](https://github.com/BabylonJS/Babylon.js/blob/b54bb0366da18e2c6c3dc3b8ac65f6af189b9b02/src/Shaders/oitFinal.fragment.fx#L12)).

Reply:
Ah, thanks for the explanation. The results look good, so 👍